core/storage/email/gae_models.py

```diff
@@ -341,7 +341,7 @@ def create(cls, user_id, thread_id):
             thread_id: str. ID of the corresponding thread.
 
         Returns:
-            str. A unique ID that can be used in 'reply-to' email address.
+            the created FeedbackEmailReplyToIdModel instance.
```

Comment:
Please follow the following format: 
Datatype. Description (starting with a capital letter, and ending with a full stop).